src/semeio/fmudesign/design_distributions.py

```diff
@@ -82,27 +82,29 @@ def _check_dist_params_uniform(dist_params: Sequence[str]) -> tuple[bool, str]:
     return status, msg
 
 
-def _check_dist_params_triang(dist_params: Sequence[str]) -> tuple[bool, str]:
+def validate_triangular_params(
```

Comment:
Should we call it `parse_triangular_params` instead, or `parse_and_validate_triangular_params`?

Then you do not need the docstring, which is unclear about what kind of convertion it is doing.

Reply:
Not sure `parse` is a good name since there's no parsing going on. But perhaps we should remove the docstring as it's not really necessary.

Reply:
It is "parsing" from str|int|float into float

Reply:
At least it is doing more than validation